### PR TITLE
Bug 1830987: Add console form class to vm wizard tab pages

### PR DESCRIPTION
**Pull request by yaacov**
When using the wizard in CNV the form inside of it takes up the available width To better align with OCP we should change this so any form in the wizard content area will be of correct width.

Ref:
http://openshift.github.io/openshift-origin-design/designs/administrator/4.0/forms/#fields

Note:
We only adjust the width for the general tab, see discussion below.

Screenshot:
Create VM wizard:
screenshot-localhost_9000-2020 05 18-09_54_24

### CODE REVIEW

**Comment by yaacov** [comment]
@matthewcarleton @suomiy @pcbailey @glekner @irosenzw please review

**Comment by suomiy** [comment]
@matthewcarleton 

Why do we have to align with the console? AFAIK there is no such complex wizard elsewhere, so there is no such big need for space in other modals. The other modals do not need left wizard sidebar with disk tables on the right for example.

Plus there is a whole screen filled when you need that much space in Create new from YAML.

**Comment by yaacov** [comment]
a - removed the form css from the none form tabs ... 
b - left it for the cloud init tab because it's a form

@suomiy @matthewcarleton  WDYT ?

[1] http://openshift.github.io/openshift-origin-design/designs/administrator/4.0/forms/#fields

**Comment by yaacov** [comment]
@suomiy @matthewcarleton 

p.s my vote is to have is all using form css (short fields), maybe it's because I have a big screen ...

**Comment by suomiy** [comment]
I guess, having it one tab could be okay, but I would still vote for consistency across tabs. 

Also it might be useful to have more space for description/Certificate and especially API/Resource errors in  VM Import flow (these could get pretty long and would be easier to read with more space).

> b - left it for the cloud init tab because it's a form

The issue with Custom Script is still present

**Comment by matthewcarleton** [comment]
@yaacov @suomiy I think we should just limit the general form and leave everything else as full width. I would say though that this only gets applied for larger viewports for our usecase. I agree with @suomiy that we are the only ones using a wizard like this so we do have a different scenario. I think we could say for large viewports we just limit the size of the forms so they don't stretch for the entire width. WDYT @yaacov ?

**Comment by yaacov** [comment]
> I think we should just limit the general form and leave everything else as full width. 

@suomiy @matthewcarleton hi, now using form class only for the "general" form all else is full width.

**Comment by suomiy** [comment]
> I think we could say for large viewports we just limit the size of the forms so they don't stretch for the entire width

> hi, now using form class only for the "general" form all else is full width.

I guess we can also limit the cloud init form to be consistent. But only the form part - not the custom script area

**Comment by yaacov** [comment]
make sense to me to :-) done.

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const CloudInitFormRows: React.FC<CloudInitFormRowsProps> = ({
     authKeys.push('');
   }
   return (
-    <>
+    <span className="co-m-pane__body co-m-pane__form">
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx:87]: not sure we should be adding additional elements, especially span for the whole list.

Can we add this to `<Form>` element conditionally depending on `isForm` variable?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx:87]: hmm, trying, sec ...

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx]: done, much nicer :+1:

**Pull request merged**